web/job_specs_controller_test.go

```diff
@@ -12,13 +12,38 @@ import (
 	"github.com/smartcontractkit/chainlink/store/models"
 	"github.com/smartcontractkit/chainlink/store/presenters"
 	"github.com/stretchr/testify/assert"
+	"log"
 )
 
+func BenchmarkJobSpecsController_Index(b *testing.B) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+
+	b.ResetTimer()
+	for n := 0; n < b.N; n++ {
+		_, resp := testJobSpecsControllerIndex(b, app)
+		assert.Equal(b, 200, resp.StatusCode, "Response should be successful")
+	}
+}
+
 func TestJobSpecsController_Index(t *testing.T) {
 	t.Parallel()
+
 	app, cleanup := cltest.NewApplication()
 	defer cleanup()
 
+	j1, resp := testJobSpecsControllerIndex(t, app)
+
+	assert.Equal(t, 200, resp.StatusCode, "Response should be successful")
+
+	var jobs []models.JobSpec
+	json.Unmarshal(cltest.ParseResponseBody(resp), &jobs)
+	assert.Equal(t, j1.Initiators[0].Schedule, jobs[0].Initiators[0].Schedule, "should have the same schedule")
+	assert.Equal(t, models.InitiatorWeb, jobs[1].Initiators[0].Type, "should have the same type")
+	assert.NotEqual(t, true, jobs[1].Initiators[0].Ran, "should ignore fields for other initiators")
+}
+
+func testJobSpecsControllerIndex(t assert.TestingT, app *cltest.TestApplication) (*models.JobSpec, *http.Response) {
```

Comment:
Same as above, we'll get more accurate results if we move the set up code out of the benchmark.